enhancements/ingress/set-replace-append-delete-deny-headers.md

```diff
@@ -0,0 +1,634 @@
+---
+title: set-replace-append-delete-deny-headers
+authors:
+  - "@miheer"
+reviewers:
+  - "@frobware"
+  - "@gcs278"
+  - "@Miciah"
+approvers:
+  - "@frobware"
+  - "@gcs278"
+  - "@Miciah"
+api-approvers:
```

Comment:
This should include either myself or @deads2k as an API approver, I see there's an API PR so we can discuss the API there, but this shouldn't be merged until the API has been approved

Reply:
done